week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>As we chatted about in class, you don't need to make arrays for the constant wind speeds. If you already have numpy arrays for the x and y directions/components of the boat speed, you can simply add any integer and transform those arrays. </p><p> </p><p>I do not recommend using rcParams. The reason being, it changes all figures globally (not individual figures) -- so unless you absolutely want all figures to be exactly the same size, then an alternative method would be:</p><pre class="ql-syntax" spellcheck="false">plt.figure(figsize=(6,6)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>